### PR TITLE
Create ABOUT.md

### PR DESCRIPTION
**Pull request by mandel01**

### CODE REVIEW

File: docs/ABOUT.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Python is an exceptionally strong language for beginners because the code is highly readable and in many cases is as natural as reading a sentence in English. Code can be written and executed from the command line, in an interaction iPython session, or in a [Jupyter](http://jupyter.org) (iPython) notebook.
```
Comment (kytrinyx) [inline docs/ABOUT.md]: Another reason Python is a fantastic language for beginners is that there are many excellent resources for it.

**Comment by kytrinyx** [comment]
I really like how approachable/conversational this description is.

**Comment by kytrinyx** [comment]
We have two PRs right now for the ABOUT.md: See also #278

**Comment by Dog** [comment]
At request of #278 I've gone ahead and merged this request first.

**Pull request merged**